examples/blocking.py

```diff
@@ -0,0 +1,31 @@
+import asyncio
+
+import aioredis
+
+
+async def blocking_commands():
+    # Redis client bound to pool of connections (auto-reconnecting).
+    redis = await aioredis.create_redis_pool(
+        'redis://localhost')
+
+    async def get_message():
+        # Redis blocking commands are NOT naturally async. They block the
+        # connection that they are using. Therefore, if you exhaust a connection
+        # pool, blocking commands could wind up blocking the event loop. To
+        # avoid this unexpected behavior, always use explicit "acquire"
+        # syntax with blocking commands.
```

Comment:
I think this explanation isn't quite right - even with the acquire syntax, you can still exhaust the connection pool if you have that many blocking commands running at once. What the acquire syntax prevents is the connection being returned to the pool until the blocking command completes - e.g. - if the lpush() command got queued on the same connection behind the brpop(), it would never run, because brpop() would never have anything to pop.

Reply:
I should add that - in my not very informed opinion - this is very much automatable in the high-level commands interface returned by create_redis_pool() and that would be better than documenting how to work around it.

Reply:
If you could suggest more accurate phrasing, I'd be happy to update this. However, considering that there was no working example of a blocking command in the docs, it seemed useful to add one instead of having to find a github issue with example code?

Reply:
Something like:

"Redis blocking commands block the connection they are on until they complete. For this reason, the connection must not be returned to the connection pool until we've finished waiting on future created by brpop().  To achieve this, 'await redis' acquires a dedicated connection from the connection pool and creates a new Redis command object using it. This object is a context manager and the connection will be released back to the pool at the end of the with block."

I think it's great that you're contributing the example here!